packages/perseus/src/strings.ts

```diff
@@ -323,12 +323,17 @@ export const strings: {
     graphKeyboardPrompt: "Press Shift + Enter to interact with the graph",
     closePolygon: "Close shape",
     openPolygon: "Re-open shape",
+    srInteractiveElements: "Interactive elements: %(elements)s",
+    srNoInteractiveElements: "No interactive elements",
+
+    // TODO(LEMS-2660): The following strings are ones that will need
+    // translation tickets after all interactive graph SR strings have
+    // been finalized. Remove this comment after the tickets have been
+    // created.
     srPointAtCoordinates: {
         context: "Screenreader-accessible description of a point on a graph",
         message: "Point %(num)s at %(x)s comma %(y)s",
     },
-    srInteractiveElements: "Interactive elements: %(elements)s",
-    srNoInteractiveElements: "No interactive elements",
 };
```

Comment:
would it be better to just not put the strings here at all? I can remove the strings from this section so that they don't get auto-flagged for translations, and I can move this comment up to the `PerseusStrings` section instead so we can remember to bring them down here later?

Reply:
Turns out that it's fine if we do it this way, since strings don't get translated until they're specifically requested anyway.